database/youtube/data-structures.json

```
@@ -1,65 +1,74 @@
-[{
+[
+  {
     "name": "Jennys Lectures CSIT",
     "description": "This channel provides free courses for DSA, C, C++, Python etc.",
     "url": "https://www.youtube.com/@JennyslecturesCSIT",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "english"
-},
-{
+  },
+  {
     "name": "CodeHelp - by Babbar",
     "description": "This channel provides free placement specific DSA course in C++.",
     "url": "https://www.youtube.com/playlist?list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "Apna College",
     "description": "This channel provides free courses for DSA, Java, C++, C etc.",
     "url": "https://www.youtube.com/@ApnaCollegeOfficial",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "CodeWithHarry",
     "description": "A all in one destination for DSA, Python, C++, JavaScript, Nodejs, Java, PHP & many more.",
     "url": "https://www.youtube.com/@CodeWithHarry/playlists",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "Luv",
     "description": "This channel provides free Competitive Programming content.",
     "url": "https://www.youtube.com/@iamluv/playlists",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "take U forward",
     "description": "One of the best channel for DSA, you will find the thought process behind every problem and not just simple algorithms.",
     "url": "https://www.youtube.com/@takeUforward/featured",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "english"
-},
-{
+  },
+  {
     "name": "Pepcoding",
     "description": "One of the best channel for DSA in Java and helpful for both students and working professionals.",
     "url": "https://www.youtube.com/@Pepcoding/featured",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "Abdul Bari",
     "description": "This channel is focused on teaching Data Structures and Algorithms. They also provide courses on different computer science subjects.",
     "url": "https://www.youtube.com/@abdul_bari",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "english"
-}
+  },
+  {
+    "name": "Anuj Bhaiya",
+    "description": "Explanations of DSA concepts in a single resource, covering both CPP and Java.",
```

[Proofreading] Seems to share the same issue with [line 52](https://github.com/rupali-codes/LinksHub/pull/587/files#r1199675189). Check my feedback there to help you.
```suggestion
    "description": "Explanations of DSA concepts are provided in this resource. It also covers both CPP and Java.",
```